### PR TITLE
Suthesh/add champion

### PR DESCRIPTION
**Pull request by suthesan-deriv**
# Description

Update footer and regulatory to include champion group ltd 

Changes:

-   Update footer and regulatory to include champion group ltd 

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/regulatory/index.js [hunk 1]
```diff
@@ -381,44 +357,79 @@ const Regulatory = () => {
...
+                        <CssGridColumn>
+                            <Desktop>
+                                <StyledHeader lh="4rem" as="h4">
+                                    {localize('Deriv (SVG) LLC')}
+                                </StyledHeader>
+                            </Desktop>
+                            <Text mt="0.8rem" mb="1.6rem" max_width="58.8rem">
                                 {localize(
-                                    'Clients from the rest of the world (except for restricted countries such as the USA, Canada, and territories such as the European Union) who wish to trade CFDs on forex and cryptocurrencies will have their MetaTrader 5 accounts opened under Deriv (FX) Ltd.',
+                                    'Deriv (SVG) LLC, Hinds Buildings, Kingstown, St. Vincent and the Grenadines (company number 273 LLC 2020).',
```
Comment (kevinw-binary) [inline src/pages/regulatory/index.js:423]: I think sentence should be in bold by using Localize

Comment (kevinw-binary) [inline src/pages/regulatory/index.js:423]: I think the  sentence  
`company number 273 LLC 2020`
should be in bold and use Localize

File: src/pages/regulatory/index.js [hunk 2]
```diff
@@ -381,44 +357,79 @@ const Regulatory = () => {
...
+                            <Text max_width="58.8rem">
+                                {localize(
+                                    'Clients from the rest of the world (except for certain countries such as the USA, Canada, and Hong Kong) will have their accounts opened under Deriv (SVG) LLC.',
                                 )}
                             </Text>
                         </CssGridColumn>
+
                         <ResponsiveGrid align="flex-start">
                             <Mobile>
                                 <StyledHeader lh="4rem" as="h4" mb="1.6rem">
```
Comment (kevinw-binary) [inline src/pages/regulatory/index.js:435]: If i compare to zeplin, the line 449 

(registration no. 108101) is removed from the sentence

Comment (kevinw-binary) [reply src/pages/regulatory/index.js:435]: From line 482 to 503, it has a slight difference within the example from zeplin, but I think the current version is better than the zeplin version, not sure which one we should follow

**Review by kevinw-binary — CHANGES_REQUESTED** [review]
Added some comment

File: src/pages/regulatory/index.js (hunk 2 above)
Comment (suthesan-deriv) [reply src/pages/regulatory/index.js:435]: I think it is better to stick to zeplin design

File: src/pages/regulatory/index.js (hunk 1 above)
Comment (suthesan-deriv) [reply src/pages/regulatory/index.js:423]: I gotta check to see if there is a pdf file for this link..thanks for pointing it out 🙏

Comment (kevinw-binary) [reply src/pages/regulatory/index.js:423]: the ` company number 273 LLC 2020` 
still missing the bold styling

**Review by kevinw-binary — CHANGES_REQUESTED** [review]
added 1 comment regarding the small bold styling issues for `company number 273 LLC 2020`

**Review by kevinw-binary — APPROVED** [review]

**Pull request closed without merging**